starter-code/index.html

```diff
@@ -1,31 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel = "stylesheet" href = ./css/main.css>
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
+<div class = "container">
+  
+  <div class="left">
+  <img class = "phones" src="./images/phones.png" alt="phones">
+  </div>
 
-  Capture and Share the World's Moments
+  <section class = "right-side">
+    <img class = "logo" src="./images/brand.png" alt="IG Brand">
+    <button class = "button" href="Instagram.com"> <img class="log in" src="./images/home.png" alt="Home button"></img>Log In</button>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+    <div class = "capture-share-box">
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+      <h2>Capture and Share</br> the World's Moments</h2>
 
-  Oh yeah, did we mention it's free?
+      <p>Instagram is a <strong>fast</strong>, <strong>beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.</p>
 
-  Download on the App Store
-  Get it on Google Play
+      <p>Take a picture or video, choose a filter to transform its look</br> and feel, then post to Instagram &mdash; it's that easy. You can</br> even share to Facebook, Twitter, Tumblr and more. It's a</br> new way to see the world.</p>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+      <p>Oh yeah, did we mention it's free?</p>
+
+  <button class = "button1" href=#button1> <img src="./images/badge-iphone.png" alt="download on the app store"></img></button>
+  <button class = "button1" href=#button2> <img class="download" src="./images/badge-android.png" alt="download on the app store"></img></button>
+
+    </div>
+
+  </section>
+</div>
+  <footer>
+    <a href=#aboutus>About Us</a>
+    <a href=#support>Support</a>
+    <a href=#blog>Blog</a>
+    <a href=#press>Press</a>
+    <a href=#api>API</a>
+    <a href=#jobs>Jobs</a>
+    <a href=#privacy>Privacy</a>
+    <a href=#terms>Terms</a>
+ 
+  </footer>
 
   &copy; 2014 Instagram
```

Comment:
Be careful, your content here is not a container (p tag, div, ect..) so it won't respond as expected ;)